File: src/Java/BehavioralPatterns/Command/FastFoodRestaurant/src/Commands/FastFoodOrder.java
package Commands;

import Models.MenuItem;

import java.util.ArrayList;
import java.util.List;

public class FastFoodOrder {
    private final List<MenuItem> fastFoodOrderMenuItems = new ArrayList<>();

    public void executeCommand(OrderCommand orderCommand, MenuItem menuItem) {
        orderCommand.execute(fastFoodOrderMenuItems, menuItem);
    }

    public void showCurrentItems() {
        for (MenuItem menuItem : fastFoodOrderMenuItems) {
            menuItem.display();
        }
        System.out.println(new String(new char[15]).replace("\0", "-"));
    }
}

File: src/PHP/BehavioralPatterns/Command/FastFoodRestaurant/Commands/FastFoodOrder.php
<?php

namespace BehavioralPatterns\Command\FastFoodRestaurant\Commands;

use BehavioralPatterns\Command\FastFoodRestaurant\Models\MenuItem;

class FastFoodOrder {
    private array $fastFoodOrderMenuItems = [];

    public function __construct() {
        $this->fastFoodOrderMenuItems = [];
    }

    public function executeCommand(OrderCommand $orderCommand, MenuItem $menuItem): void {
        $orderCommand->execute($this->fastFoodOrderMenuItems, $menuItem);
    }

    public function showCurrentItems(): void {
        foreach ($this->fastFoodOrderMenuItems as $menuItem) {
            $menuItem->display();
        }
        echo str_repeat('-', 15) . PHP_EOL;
    }
}